### PR TITLE
add new make rule for generating config files for driverkit

### PR DESCRIPTION
**Pull request by Issif**
For allowing end users to contribute more easily by adding new OS/kernels for driverkit, I created a new script `generate` called with `make`.

This script takes `TARGET_DISTRO` and `TARGET_KERNEL` as inputs for automatically create the according config files for all versions of Falco lib. The README contains all details about the prodecure and how to contribute.

For example:
By running:
```shell
 make generate -e TARGET_DISTRO=debian -e TARGET_KERNEL=5.10.0-10-amd64_1
 ```
it generates:
```shell
        config/17f5df52a7d9ed6bb12d3b1768460def8439936d/debian_5.10.0-10-amd64_1.yaml
        config/3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4/debian_5.10.0-10-amd64_1.yaml
        config/5c0b863ddade7a45568c0ac97d037422c9efb750/debian_5.10.0-10-amd64_1.yaml
```
With relevant contents:
```yaml
kernelversion: 1
kernelrelease: 5.10.0-10-amd64
target: debian
output:
  module: output/17f5df52a7d9ed6bb12d3b1768460def8439936d/falco_debian_5.10.0-10-amd64_1.ko
  probe: output/17f5df52a7d9ed6bb12d3b1768460def8439936d/falco_debian_5.10.0-10-amd64_1.o
```
```yaml
kernelversion: 1
kernelrelease: 5.10.0-10-amd64
target: debian
output:
  module: output/3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4/falco_debian_5.10.0-10-amd64_1.ko
  probe: output/3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4/falco_debian_5.10.0-10-amd64_1.o
```
```yaml
kernelversion: 1
kernelrelease: 5.10.0-10-amd64
target: debian
output:
  module: output/5c0b863ddade7a45568c0ac97d037422c9efb750/falco_debian_5.10.0-10-amd64_1.ko
  probe: output/5c0b863ddade7a45568c0ac97d037422c9efb750/falco_debian_5.10.0-10-amd64_1.o
```

### CODE REVIEW

File: driverkit/README.md [hunk 1]
```diff
@@ -73,7 +74,18 @@ make -e TARGET_DISTRO=debian stats
 ## FAQ
 
 Q: Falco doesn't find the kernel module/ eBPF probe for my OS, what do I do?
-A: Go to the `config/` folder and add your kernel/OS combination there as a YAML file, then send a PR for everyone to profit!
+A:
```
Comment (cpanato) [inline driverkit/README.md:77]: this is needed?

Comment (Issif) [reply driverkit/README.md:77]: what do yoy mean?

Comment (cpanato) [reply driverkit/README.md:77]: ahh i understood now, nvm

File: driverkit/Makefile [hunk 2]
```diff
@@ -20,6 +20,11 @@ prepare: $(addprefix prepare_,$(VERSIONS))
 publish: $(addprefix publish_,$(VERSIONS))
 publish_s3: $(addprefix publish_s3_,$(VERSIONS))
 
+generate:
+	$(foreach VERSION,$(VERSIONS),\
+    utils/generate -k ${TARGET_KERNEL} -d ${TARGET_DISTRO} -v ${VERSION}; \
```
Comment (cpanato) [inline driverkit/Makefile:25]: this maybe need indentation to be similar in line 42-43
not sure

**Review by cpanato — APPROVED** [review]
/lgtm

**Review by zuc — APPROVED** [review]
/lgtm

**Review by leogr — APPROVED** [review]
/approve

**Pull request merged**